metadata-ingestion/src/datahub/ingestion/source/sql/postgres.py

```diff
@@ -219,3 +251,14 @@ def _get_view_lineage_workunits(self) -> Iterable[MetadataWorkUnit]:
                 wu = item.as_workunit()
                 self.report.report_workunit(wu)
                 yield wu
+
+    def get_identifier(
```

Comment:
we also have this `get_db_name` thing in other sources

Why are we doing stuff differently between postgres and other sources?

Reply:
We can replace this line 

`current_database = inspector.engine.url.database`
with
`current_database = self.get_db_name(inspector)` 

Will do so.